src/App.js

```diff
@@ -1,15 +1,29 @@
 import './index.css';
+import { Route, Routes } from 'react-router-dom';
+import Home from './components/Home';
 import Calculator from './components/Calculator';
 import Quote from './components/Quote';
+import Navigation from './components/Navigation';
 
-const App = () => (
-  <>
-    <div className="App">
-      <h1 className="app-title">Math Magicians</h1>
-    </div>
-    <Calculator />
-    <Quote />
-  </>
-);
+function App() {
+  return (
+    <>
+      <Navigation />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/calculator" element={<Calculator />} />
+        <Route path="/quote" element={<Quote />} />
+      </Routes>
+    </>
+  );
```

Comment:
[OPTIONAL] When I load your app on the browser, these error pops up. Consider resolving them in order to follow best practices for indentation format.